micro_dl/inference/image_inference.py

```diff
@@ -867,7 +877,7 @@ def run_prediction(self):
                     chan_slice_meta,
```

Comment:
Right now it looks like there will be no input image if you run a 3D prediction.